market_maker_keeper/gemini_market_maker_keeper.py

```diff
@@ -93,10 +93,10 @@ def __init__(self, args: list):
         super().__init__(self.arguments, self.gemini_api)
 
     def startup(self):
-        minimum_order_size, tick_size, quote_currency_price_increment = self.gemini_api.get_rules(self.pair())
+        _, tick_size, quote_currency_price_increment = self.gemini_api.get_rules(self.pair())
```

Comment:
can we make a PR against Pyexchange so we get rid of the `_` that returns on the tuple?

I think that would be cleaner